WalletWasabi.Gui/ViewModels/StatusBarViewModel.cs

```diff
@@ -62,7 +64,7 @@ public StatusBarViewModel(string dataDir, Network network, Config config, Hosted
 			UseTor = false;
 			Tor = TorStatus.NotRunning;
 			Peers = 0;
-			BtcPrice = "$0";
+			_btcPrice = LoadingText;
```

Comment:
Why did you use the `_btcPrice` field instead of the `BtcPrice` property?

Reply:
To fix the nullable warning. Given that it's assigned in constructor it is safe wrt Avalonia binding.